### PR TITLE
:art: Video block redesign #643

### LINKED ISSUE #643: :art:  :lipstick: Video Block Redesign: block redesign
## User stories

As a user, I want the "Video Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/video-message-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/video-message-block/`.

### PR DESCRIPTION
**Pull request by dennis-mburu**
# Description
This PR redesigns the video block to enhance its usability and visual appeal

Fixes #643 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
vid
vid 2

fa-video

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/video-message-block/src/lib/components/video-block/video-block.component.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <div (click)="openVideoUploadModal()">
       <div class="video-wrapper">
         <div *ngIf="!isLoadingVideo && !videoUrl; else inputHasVideo">
-          <i id="icon" class="fas fa-play-circle"></i>
+          <img src="assets/svgs/blocks/system-uicons_video.svg" alt="">
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/video-message-block/src/lib/components/video-block/video-block.component.html:7]: use ngIf over here to switch between icons. Check out how it's been done on other blocks

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]

**Comment by dennis-mburu** [comment]
I've updated accordingly, kindly review @LemmyMwaura

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/video-block.jsplumb.ts [hunk 2]
```diff
@@ -3,28 +3,30 @@ import { BrowserJsPlumbInstance } from '@jsplumb/browser-ui';
...
   jsPlumb.addEndpoint(comp.location.nativeElement, {
     // Whether the anchor is target (Other Block -> This Block)
     target: true,
-    cssClass:"block_endpoint",
-    endpoint: "Rectangle",
-    anchor: "Left",
-    maxConnections: -1
+    cssClass: 'block_endpoint',
+    endpoint: 'Dot',
+    anchor: [0, 0.13, 0, 1],
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/video-block.jsplumb.ts:27]: leave this as 'Left'

File: libs/features/convs-mgr/stories/blocks/library/video-message-block/src/lib/components/video-block/video-block.component.ts [hunk 3]
```diff
@@ -8,6 +8,7 @@ import { BrowserJsPlumbInstance } from '@jsplumb/browser-ui';
 import { VideoMessageBlock } from '@app/model/convs-mgr/stories/blocks/messaging';
 
 import { VideoUploadModalComponent } from '../../modals/video-upload-modal/video-upload-modal.component';
+import { iconsAndTitles } from '../../../../../main/src/lib/model/icons-and-titles';
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/video-message-block/src/lib/components/video-block/video-block.component.ts:11]: Try importing this from its aliased path.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Kindly sync and resolve conflicts, then ping me.

**Comment by dennis-mburu** [comment]
Made all the required changes. Kindly review @LemmyMwaura

**Review by LemmyMwaura — APPROVED** [review]

**Pull request merged**